3V0L/hellobooks-cp4
# #159239812 Implement Authentication

#### What does this PR do?
- Allow a user to register, login and reset their password

#### Description of Task to be completed?
- Allow a user to register, login or reset their password when they have the correct credentials

#### How should this be manually tested?
- Go to the landing page, register a user
- Attempt to login with that user
- Logout
- Reset their password
- Check your email for the new password
- Use the new password to login

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/159239812

#### Screenshots
<img width="826" alt="screen shot 2018-08-20 at 13 01 33" src="https://user-images.githubusercontent.com/33119403/44334079-46c34300-a479-11e8-8700-92c866114ba0.png">

_Fig 1: Login Page_

<img width="717" alt="screen shot 2018-08-20 at 13 01 45" src="https://user-images.githubusercontent.com/33119403/44334080-475bd980-a479-11e8-9cbe-f6e6aadf7500.png">

_Fig 2: Registration Page_

<img width="765" alt="screen shot 2018-08-20 at 13 02 06" src="https://user-images.githubusercontent.com/33119403/44334081-475bd980-a479-11e8-9847-b9a9ea433729.png">

_Fig 3: Reset Password_
